src/arrayMethodReduce.js

```diff
@@ -5,8 +5,20 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let acc = initialValue;
```

Comment:
What is `acc` ? Your variable names should be meaningful. Better rename to `accumulator`.